base/server/src/com/netscape/cmscore/apps/CMSEngine.java

```diff
@@ -265,6 +267,23 @@ public void initDebug() throws Exception {
         debug.init(debugConfig);
     }
 
+    public void initStartupNotifiers() throws Exception {
+        IConfigStore snConfig = config.getSubStore("startupNotifiers");
+        String ids = snConfig.getString("list", null);
```

Comment:
The theoretical conflict is:
```
startupNotifiers.list=list
startupNotifiers.list.class=...
```

Which should work.  Bit of a strange choice to use `list` as an ID here, though :)

Do we expect multiple startup notifiers?  Not in particular.  The main reason I chose to use a container was to properly represent the presence/absense of a configure notifier in the type instead of relying on `null` checking.  So either use `Optional` for a zero-or-one, or use `List` which is is a trivial amount of additional work and supports zero-or-many notifiers.

Reply:
Yeah, this is actually consistent with other `list` parameters in `CS.cfg`, so that's fine.